### PR TITLE
Better wording and spelling

### PR DESCRIPTION
**Pull request by Mego**

### CODE REVIEW

**Comment by vihanb** [comment]
I'm sorry I cannot merge this unless the tests pass.

**Comment by Mego** [comment]
Hue hue hue

**Comment by vihanb** [comment]
-1 as this PR represents discrimination against animals. As a goat, I have hooves making it hard to type, this PR is mocking my disability making it very offensive 0/10

(but on a more serious note it's not letting me merge until the checks pass so I can't)

**Comment by schas002** [comment]
Codecov is taking ages to run :|

**Comment by vihanb** [comment]
codecov hates penguins confirmed ;_;

**Comment by schas002** [comment]
Probably, you can find a channel to contact with Codecov at https://codecov.io/support.

**Comment by vihanb** [comment]
@schas002 i was making a joke. The travis build didn't have the node dependency since I cleared the cache last. It was previously caching node_modules/ which included the babel-cli dep. The build would error and therefore codecov would never recieve the coverage report and as a result appear to taking forever.

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ addons:
       - g++-4.8
 install:
   - npm install -g codeclimate-test-reporter
+  - npm install -g babel-cli
```
Comment (vihanb) [inline .travis.yml]: add a `npm install` after this so it works

**Comment by Mego** [comment]
I think Travis broke - the latest commit isn't showing up on Travis's history

**Comment by schas002** [comment]
> @schas002 i was making a joke. The travis build didn't have the node dependency since I cleared the cache last. It was previously caching node_modules/ which included the babel-cli dep. The build would error and therefore codecov would never recieve the coverage report and as a result appear to taking forever.

Ah k :smile:

**Pull request closed without merging**